clusterctl-settings.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
I'm assuming that this isn't used anywhere?

Reply:
No. Its a CAPI development thing that you can use, see https://cluster-api.sigs.k8s.io/clusterctl/developers.html. 

The file is an incomplete configuration. I gave it a whirl and `clusterctl` errored out requiring more info. I fixed it up but other problems ensued. In any case, its not used by the project explicitly and requires a checkout of CAPI.